projects/packages/jetpack-mu-wpcom/src/features/wpcom-dashboard-widgets/wpcom-dashboard-widgets.php

```diff
@@ -48,7 +48,7 @@ function enqueue_wpcom_dashboard_widgets() {
 
 	$data = wp_json_encode(
 		array(
-			'siteName'    => get_bloginfo( 'name' ),
+			'siteName'    => htmlspecialchars_decode( get_bloginfo( 'name' ), ENT_QUOTES ),
```

Comment:
I wonder if this should use `wp_specialchars_decode` instead of `htmlspecialchars_decode`. A quick grep through Core looks like that's what they use when needing to decode entities in `get_bloginfo( 'name' )`.